gutterball/src/test/resources/logback-test.xml

```diff
@@ -10,7 +10,13 @@
   <logger name="org.hibernate.id.UUIDHexGenerator" level="ERROR"/>
   <logger name="liquibase" level="ERROR"/>
 
-  <logger name="org.candlepin.gutterball" level="INFO"/>
+<!--
+  <logger name="org.hibernate.SQL" level="DEBUG"/>
+  <logger name="org.hibernate.type" level="TRACE"/>
+-->
+
+
+  <logger name="org.candlepin.gutterball" level="DEBUG"/>
```

Comment:
In test, probably not unless it's generating a lot of noise. I don't really see it hurting anything either way, so I'm okay with whichever.